lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -833,7 +833,19 @@ class BitmapCanvas extends EngineCanvas {
   ///
   /// The text is drawn starting at coordinates ([x], [y]). It uses the current
   /// font set by the most recent call to [setCssFont].
-  void fillText(String text, double x, double y) {
+  void fillText(String text, double x, double y, {List<ui.Shadow>? shadows}) {
+    if (shadows != null) {
+      _canvasPool.context.save();
```

Comment:
i'd pull _canvasPool.context into a local and use it throughout for less code.

Reply:
Good point! Done.